KaraIOke/Services/AppEnvironment/AppEnvironmentService.cs

```diff
@@ -15,16 +15,20 @@ public AppEnvironmentService(ISearchService searchMockService, ISearchService se
     }
 
     public ISearchService SearchService { get; private set; }
+    public IPlaylistService PlaylistService { get; private set; }
 
     public void updateDependencies(bool useMockServices)
     {
         if (useMockServices)
         {
             SearchService = _searchMockService;
+            PlaylistService = new PlaylistMockService();
```

Comment:
Instantiate `PlaylistMockService` in MauiProgram and pass it to `AppEnvironmentService` through constructor, store it in private variable, same as `SearchService`.